etc/spack/defaults/packages.yaml

```diff
@@ -52,3 +52,9 @@ packages:
     permissions:
       read: world
       write: user
+  openmpi:
+    buildable: False
+    externals:
+    - spec: 'openmpi@3.1.6-gcc_9.3.0'
+      modules:
+      - 'openmpi/3.1.6-gcc_9.3.0'
```

Comment:
I assume this was accidentally added to the PR...

Reply:
Yes, sorry. I didn't specifically `git add` that file but the commit seems to have picked it up regardless. Let me whack that.

Reply:
Yeah, `git commit -a` will do that...